### PR TITLE
Add user to the group

### PR DESCRIPTION
**Pull request by samta**

### CODE REVIEW

**Comment by samta** [comment]
Hi Naman,

Raise merge request again develop branch, Can you please review.

Thanks,
Samta

File: splitwise/__init__.py [hunk 1]
```diff
@@ -280,3 +281,13 @@ def setUserArray(users, user_array):
                 else:
                     gen_key = key
                 user_array["users__" + str(count) + "__" + gen_key] = user_dict[key]
+
+    def addUserToGroup(self, add_user_request, group_id=0):
```
Comment (namaggarwal) [inline splitwise/__init__.py:285]: what is add_user_request, how would someone know what to set in that object ?

**Review by namaggarwal — CHANGES_REQUESTED** [review]

File: splitwise/__init__.py (hunk 1 above)
Comment (TurnrDev) [reply splitwise/__init__.py:285]: Hi, I'm not the guy who wrote this. Just passing by. Use splitwise and plan to contribute in future to this project.

Judging by the code and the [API docs](http://dev.splitwise.com/dokuwiki/doku.php?id=add_user_to_group), it looks like it's expecting an object which it then converts styled like this:

{ 'user_id' : 123456 }

or like this 

{ 'first_name' : 'John', 'second_name' : Smith, 'email' : 'john_smith@yahoo.com' }

Which, judging by your existing code, it's looking for a user defined in your user.py file. The only issue is @samta didn't take into account the `registration_status` and `picture` attributes of a User class object and how they would affect the post request.

Comment (namaggarwal) [reply splitwise/__init__.py:285]: Hey @JayTurnr ,

Yes I get that part, In that case ```add_user_request``` should just be called ```user```. I also don't think setting the group_id to 0 by default is right. Also ```registration_status``` and ```picture``` are not required for this request so its fine to leave them out.

Comment (namaggarwal) [reply splitwise/__init__.py:285]: Also I am looking forward to your contributions. There are lot of APIs still not implemented in this SDK and I am a bit busy lately.

**Comment by namaggarwal** [comment]
Closing as I am taking this in the new version of the API

**Pull request closed without merging**